lib/helpers.py

```diff
@@ -141,8 +168,6 @@ def read_csv(customer, audience, source, date, columns=None, revenue_event=None,
         else:
             filtered_chunk = chunk
 
-        # we are not interessted in events that do not have a group
-        filtered_chunk = filtered_chunk[filtered_chunk['ab_test_group'].isin(['test', 'control'])]
```

Comment:
Maybe it makes sense to keep this clause for `marks_and_spend_df`, but drop it for `attributions`? This way we assure, that we have at least _some_ mark information about the user, and not just blindly assume it to be a `control`

Reply:
...resolved via lines below